generators/route/index.js

```diff
@@ -58,6 +87,9 @@ module.exports = class PageGenerator extends BaseSubGenerator {
     );
   }
 
+  /**
+   * Create template.
```

Comment:
How about use something like 
`Method for "writing" phase of yeaoman generator`?